helix-view/src/tree.rs

```diff
@@ -434,6 +434,10 @@ impl Tree {
             self.focus = key;
         }
     }
+
+    pub fn full_area(&self) -> Rect {
```

Comment:
Call it either area or viewport to match terminology elsewhere

Reply:
Right, done.